mobile/bazel/envoy_mobile_dependencies.bzl

```diff
@@ -60,6 +60,8 @@ def kotlin_dependencies(extra_maven_dependencies = []):
     maven_install(
         artifacts = [
             "com.google.code.findbugs:jsr305:3.0.2",
+            # Java Proto Lite
+            "com.google.protobuf:protobuf-javalite:3.24.4",
```

Comment:
do we need this in this PR?

Reply:
Yes, otherwise we can't access the Java Proto API from Java.